src/services/authBroadcastService.ts

```diff
@@ -0,0 +1,20 @@
+export interface IAuthBroadcastMessage {
+  authenticated: boolean;
+}
+
+class AuthBroadcastService {
```

Comment:
Consider using pure function instead of class, see for example https://github.com/project-ncl/pnc-web-ui-react/blob/main/src/services/webSocketClient.ts